pkg/volume/flexvolume/driver-call.go

```diff
@@ -168,6 +169,8 @@ func NewOptionsForDriver(spec *volume.Spec, host volume.VolumeHost, extraOptions
 		options[optionReadWrite] = "rw"
 	}
 
+	options[optionPVorVolumeName] = spec.Name()
```

Comment:
Detach API uses "PVOrVolumeName". If we do not pass it during attach API, driver has no way of mapping it to back during Detach Call. Current workaround/limitation is to use the same name for actual driver backed volume name & PVName.